mlir/test/Dialect/SCF/parallel-loop-tiling-check-tile-size.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: mlir-opt %s -pass-pipeline='builtin.module(func.func(scf-parallel-loop-tiling{parallel-loop-tile-sizes=0,0}))' -split-input-file -verify-diagnostics
+
```

Comment:
nit: Such files are usually called `parallel-loop-invalid.mlir`. Other error checking unit tests can then be added over time.